scheduler/src/cook/mesos/util.clj

```diff
@@ -91,18 +91,19 @@
 (defn entity->map
   "Takes a datomic entity and converts it along with any nested entities
   into clojure maps"
-  ([entity db]
-   (entity->map (d/entity db (:db/id entity))))
-  ([entity]
-   (deep-transduce-kv (map identity) entity)))
+  ([entity & {:keys [db skip-keys]
+              :or {skip-keys #{}}}]
+   (let [entity' (if db (d/entity db (:db/id entity))
+                     entity)]
+     (deep-transduce-kv (filter (fn [[k _]] (not (contains? skip-keys k)))) entity'))))
 
 (defn job-ent->map
   "Convert a job entity to a map. This also loads the associated group and converts it to a map."
   ([job db]
    (job-ent->map (d/entity db (:db/id job))))
   ([job]
    (-> (entity->map job)
-       (assoc :group/_job (entity->map (:group/_job job))))))
+       (assoc :group/_job (entity->map (:group/_job job) :skip-keys #{:group/job})))))
```

Comment:
Might be nice to add a comment explaining why we're skipping that key

Reply:
This line is pretty confusing in general, given the `:group/_job` and `:group/job`, so some extra explanation may not hurt

Reply:
Done